bsip-0085.md

```diff
@@ -0,0 +1,49 @@
+    BSIP: 0085
+    Title: Maker order creation fee discount
+    Author: Abit More <https://github.com/abitmore>
+    Status: Draft
+    Type: Protocol
+    Created: 2019-10-13
+    Discussion: https://github.com/bitshares/bsips/issues/240
+    Worker: TBD
+
+# Abstract
+
+This BSIP proposes a protocol change so that the committee can define
+different fee rates for maker orders and taker orders.
+
+# Motivation
+
+To improve liquidity of the BitShares DEX.
+
+# Rationale
+
+The maker-taker fee model is adopted widely in centralized exchanges and
+helped them to attract trading activities.
+
+In the BitShares DEX, for each filled order, the owner of the order need to
+pay an order creation fee and conditionally a market fee. [BSIP 81](
+https://github.com/bitshares/bsips/blob/master/bsip-0081.md) described
+a way to apply the maker-taker fee model to the market fee. This BSIP
+describes a way to apply the maker-taker fee model to the order creation fee.
+
+# Specification
+
+Add a new global parameter `maker_fee_discount_percent` which can
+be updated by the committee only after the protocol upgrade.
+Initial value of that parameter is `0`.
+Valid range of that parameter is `[0, 100%]`.
+
+When a limit order with a non-zero pending fee is filled or partially filled,
+* if the order is a take order, process the pending fee as before;
+* if the order is a maker order, return
+`pending_fee * maker_fee_discount_percent` (round down) to the owner,
+then process the remaining pending fee as before.
+ Note: similar to [BSIP 26](
+https://github.com/bitshares/bsips/blob/master/bsip-0026.md), in case when
+ the order creation fee was paid in an asset which was not BTS,
+ return an amount of BTS to the asset's fee pool and
+ return an amount of that asset to the order owner.
```

Comment:
Is this from the `deferred_paid_fee` field?
How much is the amount?

Reply:
I didn't want to add the details into the specification. It's similar to BSIP26. If you think it's necessary, perhaps create a PR? Thanks.

Reply:
It isn't clear to me which is why I asked.  Therefore I cannot make a PR.  Without the clarity it will be left up to the code reviewer to interpret whether the new code is similar enough to BSIP26.

Reply:
I will update.

Reply:
Updated, please review.

Reply:
I greatly appreciate the elaboration @abitmore .  The update is clear and helpful.